packages/marko-web-gam/browser/inject-ads.vue

```diff
@@ -69,6 +69,8 @@ export default {
                   const $previous = $(this).prev();
                   if (component.detectEmbeds && $previous.attr('data-embed-type')) {
                     $(this).after(cleaned);
+                  } else if ($previous.is(':header')) {
```

Comment:
Read the performance implications of using the jQuery `:header` pseudo-selector... https://api.jquery.com/header-selector/ 

I'll. leave it up to you if you want to still use it, or use native selectors. I can go either way